lighthouse-core/audits/preload-lcp-image.js

```diff
@@ -86,22 +90,33 @@ class PreloadLCPImageAudit extends Audit {
    * @param {LH.Gatherer.Simulation.GraphNode} graph
    * @param {LH.Artifacts.TraceElement|undefined} lcpElement
    * @param {Array<LH.Artifacts.ImageElement>} imageElements
-   * @return {LH.Gatherer.Simulation.GraphNetworkNode|undefined}
+   * @return {{lcpNodeToPreload?: LH.Gatherer.Simulation.GraphNetworkNode, initiatorPath?: InitiatorPath}}
```

Comment:
could this continue to return `| undefined`, and in the call sites just destructure with ` || {}`?

Reply:
no, `lcpNodeToPreload` can be undefined in cases where `initiatorPath` won't be.